### ISSUE TITLE
Crash on launching on macOS

### ISSUE THREAD

**Issue by hajimehoshi** [hajimehoshi/ebiten#1237]
```
panic: VersionUnavailable: NSGL: Failed to create OpenGL context

goroutine 1 [running]:
github.com/hajimehoshi/ebiten/internal/uidriver/glfw.init.1()
        /Users/hajimehoshi/ebiten/internal/uidriver/glfw/ui.go:119 +0x59
exit status 2
```

**Comment by hajimehoshi** [comment]
Rebooting my machine fixed the issue. There were no other similar issues.

**Comment by hajimehoshi** [comment]
3eaa7dd0e1e8eb52da7fd8798e18d4d63923a3d6 mitigates the issue.

I run into this issue again. I'll merge this fix to 1.11.

**Comment by hajimehoshi** [comment]
See #1243: Rebooting Chrome changed the situation.
